src/expressionDescriptor.ts

```diff
@@ -461,7 +461,20 @@ export class ExpressionDescriptor {
       description = allDescription;
     } else if (!StringUtilities.containsAny(expression, ["/", "-", ","])) {
       description = StringUtilities.format(getDescriptionFormat(expression), getSingleItemDescription(expression));
-    } else if (expression.indexOf("/") > -1) {
+    } else if (expression.indexOf("/") > -1 && expression.indexOf(",") > -1) {
+      let segments: string[] = expression.split(",");
+      let segmentDescriptions: string[] = [];
+      for (let i = 0; i < segments.length; i++) {
+        segmentDescriptions.push(this.getSegmentDescription(segments[i],
```

Comment:
I like this recursive approach for handling multiple values in a segment delimited by a comma.  Once this PR is merged, I plan to circle back and do a bit of refactoring to use this approach in a few other places.